extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -102,6 +101,7 @@ export let InteractiveComponentDef;
  *    supportedBrowserState: boolean,
  *    systemUiIsVisibleState: boolean,
  *    uiState: !UIType,
+ *    viewportWarningState: boolean,
```

Comment:
I think we should rename this to `viewportSizeWarningState`, otherwise it's not clear what the state is about.

Reply:
It can actually be about the size (desktop) or the orientation (mobile)